datasets/united_states.md

```diff
@@ -44,7 +44,8 @@ parent: Data Sources
 
 #### [Center for Disease Control and Prevention](https://www.cdc.gov/)
 * [CDC 500 Cities: Local Data for Better Health](https://chronicdata.cdc.gov/500-Cities-Places/500-Cities-Local-Data-for-Better-Health-2019-relea/6vp6-wxuq): The 500 Cities Project datasets contain model-based small area estimates for 27 measures of chronic disease related to unhealthy behaviors (5), health outcomes (13), and use of preventive services (9) for the 500 largest cities in the US.
-* [CDC Wonder Compressed Mortality](https://wonder.cdc.gov/cmf-icd10.html): mortality counts for all US states and counties broken down by underlying cause of death, age, race, sex, and year. [CDC Wonder Data Terms of Service](https://wonder.cdc.gov/datause.html).
+* [CDC Wonder: Compressed Mortality](https://wonder.cdc.gov/cmf-icd10.html): mortality counts for all US states and counties broken down by underlying cause of death, age, race, sex, and year. [CDC Wonder Data Terms of Service](https://wonder.cdc.gov/datause.html).
```

Comment:
nit: two colons look a bit odd, perhaps `CDC Wonder - Compressed Mortality:` (for Natality too) ?

Reply:
done